Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -26,6 +37,17 @@
   category: Service
   group: market
 
+- type: cargoProduct
+  id: CrateVendingMachineRestockCondimentStation
+  name: Condiment Station restock crate
+  icon:
+    sprite: Objects/Specific/Service/vending_machine_restock.rsi
+    state: base
+  product: CrateVendingMachineRestockCondimentStationFilled
+  cost: 2300  #until prices are refactored, everything costs $20. This should eventually be reduced, hopefully.
```

Comment:
Can't you just adjust condiment packet price?

Reply:
I can....I am assuming there is going to need to be a major overhaul at some point though....? Lot's of things priced at $20 is wildly out of balance.

Reply:
I'm thinking they should only be like 1, maybe 2?

Reply:
1

Reply:
Oh nice, it looks like you can do decimals. For some reason I thought it would round to a whole number. so I put them at .50, Metal utensils at 3, plastic at .75

Reply:
Oh, also I didn't see your "1" comment. Let me know, I can change whatever.

Reply:
that's fine

Reply:
Man this shit is so broken. A broken plate $20. Like, you can just make bank if you rerouted the garbage disposal to cargo.